source/includes/usage-examples/run-command.c

```diff
@@ -0,0 +1,157 @@
+#include <bson/bson.h>
+#include <mongoc/mongoc.h>
+#include <stdio.h>
+
+int
+main (int argc, char *argv[])
+{
+    mongoc_client_t *client;
+    mongoc_collection_t *collection;
+    char *str;
+    bson_error_t error;
+
+    mongoc_init ();
+
+    client =
+        mongoc_client_new ("<connection string URI>");
+    collection = mongoc_client_get_collection (client, "<database name>", "collection name");
+
+    {
+        // start-execute-command
+        bson_t *command, reply;
+        bson_error_t error;
+        bool retval;
+    
+        command = BCON_NEW ("hello", BCON_INT32 (1));
+        retval = mongoc_client_command_simple (client, "admin", command, NULL, &reply, &error);
+    
+        if (!retval) {
+            fprintf (stderr, "Failed to run command: %s\n", error.message);
+        } else {
+         char *str = bson_as_canonical_extended_json (&reply, NULL);
+            printf ("%s\n", str);
+            bson_free (str);
+        }
+    
+        bson_destroy (command);
+        bson_destroy (&reply);
+        // end-execute-command
+    }
+
+    {
+        // start-execute-command-with-options
+        bson_t *command;
+        bson_t *opts;
+        bson_t reply;
+        bson_error_t error;
+        bool retval;
+
+        command = BCON_NEW(
+            "grantRolesToUser", BCON_UTF8("user011"),
+            "roles", "[", BCON_UTF8("readWrite"), "]"
+        );
+
+        opts = BCON_NEW(
+            "writeConcern", "{",
+                "w", BCON_UTF8("majority"),
+            "}"
+        );
+
+        retval = mongoc_client_command_with_opts(client, "admin", command, opts, &reply, &error);
+
+        if (!retval) {
+            fprintf(stderr, "Failed to run command: %s\n", error.message);
+        } else {
+            char *str = bson_as_canonical_extended_json(&reply, NULL);
+            printf("Command Result: %s\n", str);
+            bson_free(str);
+        }
+
+        bson_destroy(command);
+        bson_destroy(opts);
+        bson_destroy(&reply)
+        // end-execute-command-with-options
+    }
+
+    {
+        // start-run-command-example
+        bson_t *cmd;
+        bson_t *opts;
+        mongoc_write_concern_t *write_concern;
+        mongoc_read_prefs_t *read_prefs;
+        mongoc_read_concern_t *read_concern;
+        bson_t reply;
+        bson_error_t error;
+        char *json;
```

Comment:
```suggestion
        char *str;
```

Suggest renaming for consistency with other examples.